### PR TITLE
LG Static Site Update

### PR DESCRIPTION
**Pull request by lgarceau768**
### Proposed Changes
Updated the instructions for downloading the snippet.txt file to match the following acceptance criteria.

### Change Details
Adding the curl command to the static site as well as how to verify the correct type of text file on Mac and Linux

### Acceptance Criteria:

Update static site with instructions on how to download the snippet.txt file from a terminal environment

You can use this cURL to get the file from GitHub:

curl -JLO https://raw.githubusercontent.com/CMSgov/dpc-app/master/dpc-web/public/snippet.txt 
If you are using bash, zsh, or similar you can check the file type with this command (will not work in cmd or powershell)

file snippet.txt 
and this should output the following type:

snippet.txt: ASCII text, with no line terminators 

If `with no line terminators` is not present, the keys and signature will not properly register with out system.

The final instructions should be reviewed by Vera.

### CODE REVIEW

File: common/docsV1.md [hunk 1]
```diff
@@ -249,6 +249,22 @@ Public keys verify that client token requests are coming from an authorized appl
 
 **1. Download the snippet.txt file located in the DPC Portal to create a signature.**
 
+- You are also able to download the snippet.txt file using the command invocation below:
```
Comment (TiannaW24) [inline common/docsV1.md:252]: We need some kind of call out in the docs stating "if you see this error when uploading your public keys, please refer to these instructions for downloading the snippet.txt and re-generate your public and private keys..." or something of the like

**Review by mftcodes — APPROVED** [review]
that looks good to me. Has the curl, which is system agnostic, and the check for *nix systems.

**Comment by lgarceau768** [comment]
Has been deployed to staging! [Staging URL](https://stage.dpc.cms.gov/) Can be seen at this [anchor](https://stage.dpc.cms.gov/docsV1#step-three-public-keys)

File: common/docsV1.md [hunk 2]
```diff
@@ -249,6 +249,23 @@ Public keys verify that client token requests are coming from an authorized appl
...
+  ~~~
+
+**1a. Verify the type of file.**
+
+- If you are using a Mac or Linux shell you can check the file type with this command (will not work in a basic Windows environment)
+
+  ~~~
+  file snippet.txt 
+  ~~~
+  <p style="font-weight: 700;">Response <u>must yield</u> <span style="color: #2A7A3B;">snippet.txt: ASCII text, with no line terminators</span>.</p>
```
Comment (TiannaW24) [inline common/docsV1.md:266]: While I like this addition, it doesn't resolve my above comment.

Comment (lgarceau768) [reply common/docsV1.md:266]: Could you elaborate more on what you are looking for?

File: common/docsV1.md [hunk 3]
```diff
@@ -249,6 +249,22 @@ Public keys verify that client token requests are coming from an authorized appl
 
 **1. Download the snippet.txt file located in the DPC Portal to create a signature.**
 
+- If you are unable to download the snippet.txt file with a GUI you can download the file using the command invocation below:
```
Comment (vlivshin) [inline common/docsV1.md:252]: Add a comma after  "GUI".

File: common/docsV1.md [hunk 4]
```diff
@@ -249,6 +249,22 @@ Public keys verify that client token requests are coming from an authorized appl
...
+
+  ~~~
+  curl -JLO https://raw.githubusercontent.com/CMSgov/dpc-app/master/dpc-web/public/snippet.txt 
+  ~~~
+
+**1a. Verify the type of file.**
+
+- If you are using a Mac or Linux shell you can check the file type with this command (will not work in a basic Windows environment)
```
Comment (vlivshin) [inline common/docsV1.md:260]: - Add a comma after "shell". 
- Replace "(will not work in a basic Windows environment)" with "This command will not work in a basic Windows environment.".
- Add period after "command" in the first sentence.

Updated sentences are: If you are using a Mac or Linux shell, you can check the file type with this command. This command will not work in a basic Windows environment.

File: common/docsV1.md [hunk 5]
```diff
@@ -279,6 +295,8 @@ Public keys verify that client token requests are coming from an authorized appl
 
 **6. Click Add Key** to upload your public key.
 
+- <p>If you see the error stating <span style="color: #d14">"Unable to verify your public key"</span> after uploading your public key, please re-download the snippet.txt file using the command stated in <a href="#create-a-public-key-signature" class="guide_sub-link">Step 1</a>, and re-generate your public key and signature pair!</p>
+
```
Comment (vlivshin) [inline common/docsV1.md:299]: - Add comma after word "stating".
- Change "error" to "error message".
-  Make error message text black like rest of the text.
- Replace exclamation point with period.

**Review by vlivshin — APPROVED** [review]
Looks good on staging site.

**Pull request merged**